parade/ddl.sql

```diff
@@ -56,7 +59,8 @@ LANGUAGE sql VOLATILE AS $$
         num_tries = num_tries + 1,
         performance_token = gen_random_uuid(),
         action_deadline = NOW() + max_duration -- NULL if max_duration IS NULL
-    WHERE id IN (
+    WHERE id IN (-- tzahi: you may use ctid instead,faster.
+        -- tzahi: why don't you use simple update x=x where condition? you do not need to access the id PK at all
```

Comment:
I need somewhere to say `SKIP LOCKED`, otherwise I hit many conflicts: all workers looking for a particular action try to grab it, exactly one will succeed.

Reply:
Fine, Then use CTID instead of ID. Saves going through the index to a row you already retrieved. Just replace ID with ctid